### ISSUE TITLE
Instainsane not detecting passwords?

### ISSUE THREAD

**Issue by Renacentista** [thelinuxchoice/instainsane#14]
Greetings,

I just started learning Python and using Instainsane. I was very surprised at how fast Instainsane is, since I was using a different program called "Instagram-py" before, which tests the passwords at a slow rate (like one per second). However, I tried using one of the passwords in my list and Instainsane did not detect it. 

Also, how can I be sure that my IP is being changed? Aside from the program telling me that it started Tor on ports 9051 through 9055, and that it checked the connection on each port there's nothing else (when using Instagram-py it tells you "changing IP to...etc").

**Comment by Renacentista** [comment]
UPDATE: I tried the program a couple more times and it worked! It found the password in my list! However, there's a "Challenge required", and Instagram asks to enter security code cause it detected  an unusual login attempt. Any method to get rid of this? Thanks

**Comment by AbhinavAbhyakhar** [comment]
![image](https://user-images.githubusercontent.com/42883801/60400053-a16b0800-9b3c-11e9-8100-63d34086966d.png)
still not detecting

**Comment by RTPi** [comment]
Renacentista, did you change anything between attempts?

**Comment by AlbinIbraimi** [comment]
It's not working for me... :( 
Didn't detect the password from the wordlist, and in the end of the wordlist it just stop working and didn't say anything about detection like:
"password detection failed/succeed"

**Comment by jaylampe** [comment]
> ![image](https://user-images.githubusercontent.com/42883801/60400053-a16b0800-9b3c-11e9-8100-63d34086966d.png)
> still not detecting

I had that too and I still have that. Sometimes I just close the terminal and try later or I do it multiple times and it always ends up working.

**Comment by muhammadubaid3240** [comment]
yeah i also used the 285 no password on my instagram and then put my insta username to instainsane but it didnt work
